webdriver/tests/find_element_from_shadow_root/find.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from tests.support.asserts import assert_success
+
+
+def get_shadow_root(session, element):
```

Comment:
The tests in this file are testing `Find Element`. So for cases where we have to retrieve the shadow root, the appropriate webdriver client implementation (`element.shadow_root`) should be used.

Reply:
Done in 94b8624